docs: Migrate from README to MkDocs documentation (#73)
operate-first/service-catalog: docs/index.md

# Service Catalog

> This is the service catalog for the [operate-first][3] community cloud.

Reviewer: ```suggestion
This is the service catalog for the [Operate First Community Cloud][3].
```

Author: Resolved in the latest commit.

## Architecture

> The service catalog is implemented using [backstage][4]. This instance of backstage is built using two basic packages, `app` and `backend`. The `app` packages serves as the frontend and `backend` package as the backend. These two applications then communicate with each other to create the service catalog.

Reviewer: ```suggestion
The service catalog is implemented using [Backstage][4]. There are 2 packages present in the repo:

1. `app` package for front end
2. `backend` package for back end
```

Author: Resolved in the latest commit.

## Catalog items

> Backstage can utilize a separate location (for example a GitHub repository) for the catalog component definitions. More information on how to configure the entities can be found in the [configuring entities section](configuring-entities.md).

Reviewer: ```suggestion
Backstage can utilize a separate location (for example a GitHub repository) for the catalog entity definitions. More information on how to configure the entities can be found in the [configuring entities section](configuring-entities.md).
```

Author: Resolved in the latest commit.

> Component definition files for this service catalog are located [here][1]. If you want to edit them, make a pull request in that repository.

Reviewer: ```suggestion
Entity definition files for this service catalog are located [here][1]. If you want to edit them, make a pull request in that repository.
```

Author: Resolved in the latest commit.

> There is an `example` folder located that only contains catalog items dedicated to local development. Switching between local and remote sources can be configured in `app-config.local.yaml` in the `catalog.locations` key.

Reviewer: ```suggestion
There is an `example` folder located on this repository, that only contains catalog items dedicated to local development. Switching between local and remote sources can be configured in `app-config.local.yaml` in the `catalog.locations` key.
```
